maps/southern_cross/structures/closets/misc_vr.dm

```diff
@@ -26,6 +26,7 @@
 		/obj/item/stack/marker_beacon/thirty,
 		/obj/item/weapon/material/knife/tacknife/survival,
 		/obj/item/weapon/material/knife/machete/deluxe,
+		/obj/item/weapon/gun/energy/frontier/locked/carbine,
```

Comment:
Should remove one gun from gun locker since adding another starting gun to the department to avoid inflating weaponcount